### PR TITLE
Adds note about restarting managed instances 

### PR DESCRIPTION
**Pull request by donavonelli**

### CODE REVIEW

File: content/departments/product-engineering/engineering/cloud/delivery/managed/operations.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ You can then use regular Docker commands (e.g. `docker exec -it $CONTAINER sh`)
 2. `cd` to the `/deployment/docker-compose` folder and run:
 3. `docker compose restart sourcegraph-frontend-0 sourcegraph-frontend-internal`
 
+> NOTE: Only restart a managed instance AFTER coordinating with the customer involved and its deemed necessary
```
Comment (virginiaulrich) [inline content/departments/product-engineering/engineering/cloud/delivery/managed/operations.md]: Tiny grammar correction -- "its" should be changed to "it's" (with the apostrophe

**Review by virginiaulrich — CHANGES_REQUESTED** [review]
Slight grammar change and then if you all feel this change captures what was shared in the Slack thread, I am good with this!

File: content/departments/product-engineering/engineering/cloud/delivery/managed/operations.md (hunk 1 above)
Comment (donavonelli) [reply content/departments/product-engineering/engineering/cloud/delivery/managed/operations.md]: gotcha i will fix that now

**Review by virginiaulrich — APPROVED** [review]
Thank you!

**Pull request closed without merging**